detox/test/e2e/c-stressful-use.js

```diff
@@ -0,0 +1,41 @@
+const MULTI_TEST_COUNT = 35
+
+describe.only('Stressful Use', function() {
+  describe('Stress on React Bridge', function() {
+    beforeEach(function (done) {
+      simulator.reloadReactNativeApp(done)
+    })
+
+    it('Should say hello after tap in conditions of Stressful Bridge', function() {
```

Comment:
1. naming convention for "Should" is "should"

2. change to "should handle click during conditions of stressful bridge"